src/Microsoft.ML.Data/Commands/CrossValidationCommand.cs

```diff
@@ -57,8 +57,9 @@ public sealed class Arguments : DataCommand.ArgumentsBase
             [Argument(ArgumentType.LastOccurenceWins, HelpText = "Column to use for stratification", ShortName = "strat", SortOrder = 7)]
             public string StratificationColumn;
 
-            [Argument(ArgumentType.LastOccurenceWins, HelpText = "Columns with custom kinds declared through key assignments, for example, col[Kind]=Name to assign column named 'Name' kind 'Kind'", ShortName = "col", SortOrder = 10)]
-            public KeyValuePair<string, string>[] CustomColumn;
+            [Argument(ArgumentType.LastOccurenceWins, HelpText = "Columns with custom kinds declared through key assignments, for example, col[Kind]=Name to assign column named 'Name' kind 'Kind'",
+                Name = "CustomColumn", ShortName = "col", SortOrder = 10)]
```

Comment:
>col [](start = 52, length = 3)

still, need to add customColumns as valid command name:
Name = "CustomColumn", ShortName = "col, CustomColumns"